homework2/fastq_filtrator.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python
+# coding: utf-8
+def gc_filtering(read, gc_bounds=(0, 100)):
+    gc_num = len([GC for GC in read[1] if GC == "G" or GC == "C"])
+    read_len = len(read[1])
+    gc_content = 100 * gc_num / read_len
+    if gc_bounds[0] <= gc_content <= gc_bounds[1]:
+        return "passed"
+    else:
+        return "rejected"
```

Comment:
Упростим и будем использовать логический тип
```suggestion
    return gc_bounds[0] <= gc_content <= gc_bounds[1]:
```